interp/interpreter.go

```diff
@@ -155,7 +155,7 @@ func (r *runner) run(fn *function, params []value, parentMem *memoryView, indent
 				// which case this call won't even get to this point but will
 				// already be emitted in initAll.
 				continue
-			case callFn.name == "(reflect.Type).Elem" || strings.HasPrefix(callFn.name, "runtime.print") || callFn.name == "runtime._panic" || callFn.name == "runtime.hashmapGet":
+			case strings.HasPrefix(callFn.name, "runtime.print") || callFn.name == "runtime._panic" || callFn.name == "runtime.hashmapGet":
 				// These functions should be run at runtime. Specifically:
 				//   * (reflect.Type).Elem is a special function. It should
 				//     eventually be interpreted, but fall back to a runtime call
```

Comment:
This comment looks out of date.

Reply:
Thank you for spotting that! I've removed it (it's basically a TODO that's now done).